kdc/process.c

```diff
@@ -202,9 +203,11 @@ _kdc_audit_trail(kdc_request_t r, krb5_error_code ret)
     }
     kvbuf[j] = '\0';
 
-    kdc_log(r->context, r->config, 3, "%s %s %s %s %s%s",
-	    r->reqtype, retval, r->from, r->cname,
-	    r->sname, kvbuf);
+    kdc_log(r->context, r->config, 3, "%s %s %d %s %s %s%s",
+	    r->reqtype, retval, ret, r->from,
```

Comment:
I thought you would only add the integer ret when we don't have a known string...

Reply:
Will do.